components/UserAvatar.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div class="user-avatar">
+    <transition
+      name="lc-transition-default"
+      mode="out-in"
+    >
+      <img
+        v-if="user.avatar"
+        :src="user.avatar"
+      >
+      <div
+        v-else
+        class="user-avatar__placeholder--image"
+      />
+    </transition>
+
+    <transition
+      name="lc-transition-default"
+      mode="out-in"
+    >
+      <nuxt-link
+        v-if="user.displayName"
+        :to="{ name: 'id', params: { id: user.id } }"
```

Comment:
using `nuxt-link` in here will caused unwanted effect in `like-button` repo
i.e. redirecting to `like link` page
not sure whats best way to handle this though...

Reply:
confirm with kiu, open a new window would be good enough

Reply:
會唔會有 case 係，將來 like.co 其它位唔係咁用，想真係 nuxt link？
i.e. should it be configurable, or is this part of the purpose for this component now

Reply:
i think this component at this point is made for likecoin-button;
if like.co could use this in the future then it can be totally changed to nuxt-link?